plugins/modules/identity/sap_user.py

```diff
@@ -0,0 +1,502 @@
+        required: true
+        type: str
+    host:
+        description: The required host for the SAP system. Can be either an FQDN or IP Address.
+        required: true
+        type: str
+    sysnr:
+        description:
+          - The system number of the SAP system.
+          - You must quote the value to ensure retaining the leading zeros.
+        default: '00'
+        type: str
+    client:
+        description:
+          - The client number to connect to.
+          - You must quote the value to ensure retaining the leading zeros.
+        default: '000'
+        type: str
+    username:
+        description:
+          - The username.
+        type: str
+        required: true
+    firstname:
+        description:
+          - The Firstname of the user in the SAP system.
+        type: str
+        required: false
+    lastname:
+        description:
+          - The lastname of the user in the SAP system.
+        type: str
+        required: false
+    email:
+        description:
+          - The email address of the user in the SAP system.
+        type: str
+        required: false
+    password:
+        description:
+          - The password for the user in the SAP system.
+        type: str
+        required: false
+    useralias:
+        description:
+          - The alias for the user in the SAP system.
+        type: str
+        required: false
+    user_type:
+        description:
+          - The type for the user in the SAP system.
+          - C('A') Dialog user, C('B') System User, C('C') Communication User, C('S') Service User, C('L') Reference User
+          - Must be uppercase
+        type: str
+        required: false
+        default: 'A'
+        choices: ['A', 'B', 'C', 'S', 'L']
+    company:
+        description:
+          - The specific company the user belongs to.
+          - The company name must be available in the SAP system.
+        type: str
+        required: false
+    profiles:
+        description:
+          - Assign profiles to the user.
+          - Should be uppercase
+          - for example C('SAP_NEW') or C('SAP_ALL')
+        type: list
+        elements: str
+        default: ['']
+        required: false
+    roles:
+        description:
+          - Assign roles to the user.
+        type: list
+        elements: str
+        default: ['']
+        required: false
+
+requirements:
+    - pyrfc >= 2.4.0
+author:
+    - Rainer Leber (@rainerleber)
+'''
+
+EXAMPLES = r'''
+# Create a user
+- name: Create SAP User
+  community.sap.sap_user:
+    conn_username: 'DDIC'
+    conn_password: 'Test123'
+    host: 192.168.1.150
+    sysnr: '01'
+    client: '000'
+    state: present
+    username: ADMIN
+    firstname: first_admin
+    lastname: last_admin
+    email: admin@test.de
+    password: Test123456
+    useralias: ADMIN
+    company: DEFAULT_COMPANY
+    roles:
+      - "SAP_ALL"
+
+# Force change user
+- name: Create SAP User
+  community.sap.sap_user:
```

Comment:
Purely cosmetical: name of the task collides with comment (here and in the following tasks).